README.md

```diff
@@ -29,9 +29,9 @@ composer require "dnadesign/silverstripe-populate:dev-master"
 In your application `mysite/_config.yml` file, specify the YAML fixtures you 
 want to load.
 	
-	Populate:
+	DNA\Populate\Populate:
```

Comment:
@madmatt We use DNADesign for the namespace of other modules - can we keep that consistent?

Reply:
Oh yeah sure, I meant to check that actually and totally forgot! Will update shortly and send it through.

Reply:
Have updated now :)